src/SdkCommon/ClientRuntime/ClientRuntime/RestException.cs

```diff
@@ -12,7 +12,7 @@ namespace Microsoft.Rest
 #if FullNetFx
     [Serializable]
 #endif
-    public class RestException : Exception
+    public class RestException : Exception, IRestException
```

Comment:
@dsgouda do not change existing RestException.
This change can be made during breaking change cycle.